### PR TITLE
train_test_split test_size and shuffle parameters

### PR DESCRIPTION
**Pull request by rafa-rod**
The size of test size will be chosen by user. In case of time series, shuffle=False is required, so it is import the user change it.

### CODE REVIEW

**Comment by csinva** [comment]
Thanks, nice addition!
